python/doc/examples/calibration/calibration_logistic.ipynb

```diff
@@ -69,48 +76,71 @@
     "- the initial U.S. population $y_0$,\n",
     "- the parameters $a$ and $b$.\n",
     "\n",
-    "Indeed, Pearl and Reed [2] estimated the parameters $a$ and $b$ using the U.S. census data from 1790 to 1910 while we have the data up to 2000. Moreover, the method used by Pearl and Reed to estimate the parameters could be improved; they only used 3 dates to estimate the parameters instead of using least squares, for example. Finally, Pearl and Reed did not provide confidence intervals for the parameters $a$ and $b$. \n",
+    "Indeed, Pearl and Reed [2] estimated the parameters $a$ and $b$ using the U.S. census data from 1790 to 1910 while we have the data up to 2000. Moreover, the method used by Pearl and Reed to estimate the parameters could be improved; they only used 3 dates to estimate the parameters instead of using least squares, for example. Finally, Pearl and Reed did not provide confidence intervals for the parameters $a$ and $b$. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
     "\n",
     "## Normalizing the data\n",
     "\n",
     "The order of magnitude of $a$ and $b$ are very different. In order to mitigate this, we consider the parameter $c$ as the logarithm of $b$:\n",
+    "\n",
     "$$\n",
     "c = \\log(b).\n",
     "$$\n",
     "\n",
     "This leads to the value:\n",
+    "\n",
     "$$\n",
-    "c = \\log(1.5887 \\times 10^{-10}) = -22.58.\n",
+    "c = \\log\\left(1.5887 \\times 10^{-10}\\right) = -22.58.\n",
     "$$\n",
     "\n",
     "The order of magnitude of the population is $10^6$. This is why we consider the normalized population in millions:\n",
+    "\n",
     "$$\n",
     "z(t) = \\frac{y(t)}{10^6}\n",
     "$$\n",
     "\n",
-    "for any $t\\in[t_0, t_{final}]$.\n",
-    "\n",
+    "for any $t\\in[t_0, t_{final}]$."
```

Comment:
You may want to define $z_0 := z(0)$ here.

Reply:
Fixed.